### PR TITLE
vocab list

### PR DESCRIPTION
**Pull request by nathanallen**
This is a good start (from 19-20).

### CODE REVIEW

**Comment by justincastilla** [comment]
Great list.

**Comment by justincastilla** [comment]
Should I merge?

File: vocab.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Vocab / Jargon
+## Talking the Talk
+
+####Week 1
+* Shell (Computing)
+* command line interface
+* G.U.I.
```
Comment (tgaff) [inline vocab.md]: should we add pronunciation tips for the weird ones?  e.g. "gooey"

File: vocab.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+* bash
+* directory
+* Rubber Duck Debugging
+* UNIX
+* path / $PATH
+* UNIX Commands
+  * `man`
+  * `--help`
+  * Symbolic Link
```
Comment (tgaff) [inline vocab.md]: will they see this in their time at GA?

File: vocab.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+* UNIX Command Options
+* R.E.P.L.
+* Side Effect (Programming)
+* return value
+* reference type
```
Comment (tgaff) [inline vocab.md]: does `reference type` come up ever?

**Pull request merged**